main/telemetry.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const { InfluxDB } = require('@influxdata/influxdb-client')
+
+const client = new InfluxDB({
+  url: 'https://eu-central-1-1.aws.cloud2.influxdata.com',
+  // station-anonymous-write
+  token: '0fZyu9zjDvYlaNfOeuwgnQoUI0VcSzeYDpnOLjQyr30mz-Plqels5JHEwgKRbtCcDJbQmv62VnOV_FsZVxgoow=='
+})
+
+const writeClient = client.getWriteApi(
+  'Filecoin Station', // org
+  'station', // bucket
+  'ns' // precision
+)
+
+setInterval(() => {
+  writeClient.flush()
```

Comment:
What is the error-handling story for `flush()` - can it ever fail? We don't want to trigger unhandled rejection or uncaught error when the Station cannot reach InfluxDB.

Reply:
Great catch, I added a `.catch` https://influxdata.github.io/influxdb-client-js/influxdb-client.writeapi.flush.html